copter/source/docs/weathervaning.rst

```diff
@@ -0,0 +1,73 @@
+.. _weathervaning:
+
+Weathervaning and Wind Hold
+===========================
+
+Flying a Traditional Helicopter, or some Multicopters, in significant levels of wind can present a
+challenge. The issue is that any flat plate surface, such as a Traditional Helicopter's tail fin and boom, offers a lot of surface area for the wind to interact with. That can lead to a
+reduction in attitude and position control and high motor and ESC
+load.
+
+To reduce the impact of wind when flying in VTOL modes the ArduPilot code supports active yaw axis weathervaning which can greatly reduce the impact of wind on
+VTOL flight by keeping the vehicle pointed into the wind and reducing
+the area of the wing exposed to the wind.
+
+Active Weathervaning
+--------------------
+
+Active weathervaning acts to turn the aircraft into the
+wind when flying in GUIDED and/or AUTO position-controlled VTOL modes. You can enable
+active weathervaning by setting the :ref:`WVANE_ENABLE <WVANE_ENABLE>`
+parameter to a non-zero value. The default is not to use active
```

Comment:
You also have to enable it per mode. Just the global enable dons't do anything.

Reply:
thats the next statement, but I will add it here also